### PR TITLE
Update the mentee apply page content

### LINKED ISSUE #157: Update the mentee apply page content
**Is your feature request related to a problem? Please describe.**
- We need to update the mentee apply view 

**Describe the solution you'd like**
- Create a google doc add it as the template
- Get the google doc link from the user

image

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

The purpose of this PR is to fix #157

## Goals

- Update the description and add a new button to download the template in the mentee application.

## Approach

- Create a new constant to the application link in `constants.ts`.
- Change the description in the mentee application
- Add a new button to the mentee application and link to the newly created constant.

## Screenshots

scholarX

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
- OS: macOS BigSur
- IDE: IntelliJ IDEA
- Browser: Safari

### CODE REVIEW

File: src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx [hunk 1]
```diff
@@ -183,18 +183,28 @@ function MenteeApplication() {
...
                   >
+                    <Row>
+                      <Button
+                        className={styles.button}
+                        type="primary"
+                        icon={<DownloadOutlined />}
+                        href={APPLICATION_TEMPLATE}
+                      >
+                        Download Template
+                      </Button>
```
Comment (Gravewalker666) [inline src/scenes/Home/scenes/RequestMentors/scenes/MenteeApplication/index.tsx]: Shall we make this link open in a new tab?

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: src/constants.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export const API_URL = 'http://localhost:8080/api';
 export const AUTH_URL = 'http://localhost:8080/login';
 export const LOGOUT_URL = 'http://localhost:8080/logout';
+export const APPLICATION_TEMPLATE = 'www.application-template-link.com';
```
Comment (anjula-sack) [inline src/constants.ts]: let's add a google doc link right now instead of  changing it later @Gravewalker666

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/constants.ts (hunk 2 above)
Comment (Gravewalker666) [reply src/constants.ts]: Yeah, we could do that, @jayasanka-sack can you create a doc with the sef email and send a view-only share link to kumuditha?

**Comment by kumuditha-udayanga** [comment]
PR is been updated, please review :), @anjula-sack @Gravewalker666

**Review by anjula-sack — APPROVED** [review]
Thank you! @kumuditha-udayanga

**Review by Gravewalker666 — APPROVED** [review]
Great Job @kumuditha-udayanga

**Pull request merged**